spark/dl/src/main/scala/com/intel/analytics/bigdl/nn/keras/Topology.scala

```diff
@@ -154,33 +137,37 @@ class Sequential[T: ClassTag](val stopInferShape: Boolean = false)
    * @param module module to be add
    * @return this container
    */
-  override def add(module: AbstractModule[_ <: Activity, _ <: Activity, T]): this.type = {
+  def add(module: AbstractModule[_ <: Activity, _ <: Activity, T]): this.type = {
     if (frozen) {
       throw new RuntimeException(
         "This Sequential has been frozen, as it has been added into other container")
     }
     if (module.isInstanceOf[Sequential[T]]) {
       module.asInstanceOf[Sequential[T]].frozen = true
     }
-    Util.excludeNotKeras[T](Seq(module))
-    if (!stopInferShape) {
-      triggerBuilding(module)
-    }
-    modules += module.asInstanceOf[AbstractModule[Activity, Activity, T]]
+    validateInput[T](Seq(module))
+
+    triggerBuilding(module)
+
+    labor.asInstanceOf[TSequential[T]].modules +=
+      module.asInstanceOf[AbstractModule[Activity, Activity, T]]
     this
   }
+
+  override def computeOutputShape(inputShape: Shape): Shape = {
```

Comment:
Will this be ever used?

Reply:
It would be used when `labor` of a KerasLayer is `KSequential` as the default implementation for `computeOutputShape` is `labor.computeOutputShape`.

Reply:
`labor` can be a torch layer (e.g., Linear for Dense), and has no `computeOutputShape` or `inferShape` deined, yes?

In addition, it's somewhat confusing to call `InferShape` in `KerasLayer.build`: `InferShape` should be called when the output shape needs to be inferred; however, for `KerasLayer`, its labor is already built, and therefore it should call `labor.getOutputShape`  instead.